public/js/sodashboard.js

```diff
@@ -353,23 +359,33 @@ db.get('_local/user').then(function(data) {
       .on('error', app.onSyncError);
   }).on('error', app.onSyncError);;
 
-
-  // parse the hash
-  if (window.location.hash && window.location.hash !== '#') {
-    var hash = window.location.hash.replace(/^#/,'');
-    if (hash === 'unassigned') {
-      app.unAssignedTickets();
-    } else if (hash === 'profile') {
-      app.profileEditor();
-    } else if (hash === 'mytickets') {
-      app.myTickets();
-    } else if (hash.match(/^edit/)) {
-      var match = hash.match(/[0-9]+$/);
-      if (match) {
-        app.edit(match[0]);
+  // if user has an incomplete profile, take them to their profile page
+  // get profile from remote db because sync is probably not yet complete
+  var rdb = new PouchDB(url);
+  rdb.get(app.loggedinuser["_id"]).then(function (userdata) {
+    if (!userdata.user_id || !userdata.user_name || !userdata.so_id) {
```

Comment:
should `user_id` and `user_name` be checked? they are obtained automatically from slack login and are *read only* (user has no ability to edit them- even if they are blank)